### PR TITLE
install kernelspec as part of the package

### PR DESCRIPTION
**Pull request by minrk**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,12 +10,16 @@ source:
   md5: 971eee85d630eb4bafcd52531c79673f
 
 build:
-  number: 0
-  script: python setup.py install
+  number: 1
+  script:
+    - python setup.py install
+    - python -m ipykernel install --prefix $PREFIX
```
Comment (pelson) [inline recipe/meta.yaml]: I did not know you could do a list of lines here. 👍 

Only downside - ```$PREFIX``` isn't going to work with Windows (```%PREFIX%```). I don't really have a workaround for that other than duplicating the line and putting in selectors 😕

Comment (minrk) [inline recipe/meta.yaml]: I don't suppose there's a generic "environment variable" deal here?

Comment (minrk) [inline recipe/meta.yaml]: > I did not know you could do a list of lines here

I didn't either until I tried it here.

**Comment by pelson** [comment]
If you could quickly modify your selectors, you are good to merge (without needing to wait for CI).

**Comment by minrk** [comment]
Sorry, updated the PR and promptly went home yesterday. Should be okay, now.

**Pull request merged**